parser_test.go

```diff
@@ -832,6 +832,45 @@ func TestParseRule(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "flowbits",
+			rule: `alert http $HOME_NET any -> $EXTERNAL_NET any (msg:"Flowbits test"; flow:to_server,established; content:"epaq"; http_uri; flowbits:set,Citrix_AGAX; flowbits:noalert; classtype:attempted-user; sid:2803611; rev:4;)`,
+			want: &Rule{
+				Action:   "alert",
+				Protocol: "http",
+				Source: Network{
+					Nets:  []string{"$HOME_NET"},
+					Ports: []string{"any"},
+				},
+				Destination: Network{
+					Nets:  []string{"$EXTERNAL_NET"},
+					Ports: []string{"any"},
+				},
+				SID:         2803611,
+				Revision:    4,
+				Description: "Flowbits test",
+				Tags: map[string]string{"flow": "to_server,established", "classtype": "attempted-user"},
+				Contents: Contents{
+					&Content{
+						Pattern:      []byte("epaq"),
+						Options: []*ContentOption{
+							&ContentOption{"http_uri", ""},
+						},
+					},
+					
+				},
+				Flowbits: []*Flowbit{
+					&Flowbit{
+						Action: "set",
+						Value: "Citrix_AGAX",
+					},
+					&Flowbit{
+						Action: "noalert",
+						Value: "",
+					},
+				},
+			},
+		},
```

Comment:
can you add an error test case down near line 901 to test invalid flowbits returning error?

something like alert tcp $HOME_NET any -> $EXTERNAL_NET any (msg:"bad flowbit"; flowbits:invalid_action; ...)